script/puppeteer-tests.js

```diff
@@ -33,6 +46,12 @@ describe("UI tests", function () {
     server.close();
   });
 
+  const handlePageErrors = function (page) {
+    page.on("pageerror", (err) => {
```

Comment:
should we exit with error when this happens?

Reply:
I think it’s OK to not for now.